README.md

```diff
@@ -63,9 +63,13 @@ refresh your browser.
 Previewing Your Changes Online
 ---
 
-We have a special script to use to publish your changes using the `gh-pages`
-branch of your fork of ev3dev.github.io. If you have SSH setup for your GitHub
-account, simply run`./publish <gh-user>` where `<gh-user>` is your actual GitHub
+We have a special ruby script to use to publish your changes using the `gh-pages`
+branch of your fork of ev3dev.github.io. 
+
+It requires you to install Ruby (see previous section), but Jekyll is not needed.
```

Comment:
Technically, the `bundler` gem is not needed either.

Reply:
Why are the dependencies not needed? The publish script just publishes the most recent local build, so you need to run a build as usual first.

Reply:
Ah, you are right. I had forgotten. I was thinking that jekyll ran on GitHub just like when we publish the real website.

So, please disregard my previous comment. So, I suppose that we should just say that "after you preview locally, you can publish it for other to see using the `./publish.rb` script"

Reply:
Thanks for the info, worked it in.